backend-tests/tests/test_api_endpoints.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+import glob
+import json
+import logging
+import os
+import pytest
+import requests
+import subprocess
+import tempfile
+import yaml
+
+from testutils.api.client import GATEWAY_HOSTNAME
+
+
+logging.basicConfig(format="%(asctime)s %(message)s")
+logger = logging.getLogger("test_decomission")
+logger.setLevel(logging.INFO)
+
+
+REPO_TO_ENV_VARIABLE = {
+    "auditlogs": "AUDITLOGS_REV",
+    "deployments": "DEPLOYMENTS_REV",
+    "deployments-enterprise": "DEPLOYMENTS_ENTERPRISE_REV",
+    "deviceauth": "DEVICEAUTH_REV",
+    "deviceconfig": "DEVICECONFIG_REV",
+    "deviceconnect": "DEVICECONNECT_REV",
+    "inventory": "INVENTORY_REV",
+    "inventory-enterprise": "INVENTORY_ENTERPRISE_REV",
+    "tenantadm": "TENANTADM_REV",
+    "useradm": "USERADM_REV",
+    "useradm-enterprise": "USERADM_ENTERPRISE_REV",
+    "workflows": "WORKFLOWS_REV",
+    "workflows-enterprise": "WORKFLOWS_ENTERPRISE_REV",
+}
```

Comment:
:+1: 

But we need to make a note in [mender-qa Documentation](https://github.com/mendersoftware/mender-qa/blob/master/Documentation/adding-new-repository.md) to update this list and `REPOS` below if a new repo being added has APIs (of just if it is a backend repository, which will do most of the times)

Reply:
https://github.com/mendersoftware/mender-qa/pull/519